src/deserialize_sequence.cc

```diff
@@ -353,7 +359,16 @@ void load_step_setup_script(const std::filesystem::path& folder, Sequence& seque
     {
         std::string line;
         while(std::getline(stream, line, '\n'))
-            step_setup_script += (line + '\n');
+        {
+            auto keyword = gul14::trim_left_sv(line);
+
+            if (gul14::starts_with(keyword, "-- maintainers:"))
+                extract_maintainers(keyword.substr(15), sequence);
```

Comment:
The `extract_maintainers()` function is doing so little that the code might actually be clearer without it. But that is a matter of taste, I guess.